simapp/v2/simdv2/cmd/root_di.go

```diff
@@ -140,3 +223,13 @@ func ProvideClientContext(
 
 	return clientCtx
 }
+
+func needsApp(cmd *cobra.Command) bool {
+	if cmd.Annotations["needs-app"] == "true" {
```

Comment:
This feels a bit meh tbh, unless it is very good documented.

Reply:
also my least favorite aspect.  could also hard code subcommands e.g. `start` `genesis export`

Reply:
then we need to do the same for snapshots as well, so not sure. At least we can hardcode start for sure